detection_rules/packaging.py

```diff
@@ -460,6 +466,39 @@ def generate_xslx(self, path):
         doc.populate()
         doc.close()
 
+    def _generate_registry_package(self, save_dir):
+        """Generate the artifact for the oob package-storage."""
+        from .schemas.registry_package import get_manifest
+
+        assert self.registry_data
+
+        registry_manifest = get_manifest(self.registry_data['format_version'])
+        manifest = registry_manifest.Schema().load(self.registry_data)
+
+        package_dir = Path(save_dir).joinpath(manifest.version)
+        docs_dir = package_dir.joinpath('docs')
+        rules_dir = package_dir.joinpath('kibana', 'rules')
```

Comment:
are you a fan of this?
```suggestion
        docs_dir = package_dir / 'docs')
        rules_dir = package_dir / 'kibana' / 'rules'
```

Reply:
as slick as that is, I have been hesitant to adapt because there are often times where `joinpath` is _needed_ or more obvious and it feels weird to mix the two syntaxes so close together, even though for simple examples like this it is obvious.

On the other hand, if that is the style we want to adapt, I am ok with it: `/` where possbile `joinpath` where needed or where it scopes the code better